docs/tutorial/7-make-terminal-interactive.md

```diff
@@ -0,0 +1,255 @@
+---
+title: &title Make terminal interactive
+description: &description Your Express is up and running and we connected a terminal to the output of the WebContainer processes. Let's make the terminal more interactive allowing the user to run it's own commands.
+head:
+  - ['meta', {property: 'og:title', content: *title}]
+  - ['meta', {property: 'og:image', content: 'https://webcontainers.io/img/og/tutorial-5_editing_a_file_updating_the_iframe.png'}]
+  - ['meta', {name: 'twitter:title', content: *title}]
+  - ['meta', {name: 'twitter:description', content: *description}]
+---
+# Make terminal interactive
+
+The terminal that we setup in the previous step doesn't accept user input, it's currently just a view to see the logs with proper formatting. Wouldn't it be nice if we could convert this experience to a real terminal which allows you to install other packages, and run different commands? Let's get started!
+
+## 1. Remove code
+
+Isn't the best feeling as a developer that you can remove code which isn't necessary anymore? You can remove both the `installDependencies` and `startDevServer` methods. The only thing that we want to keep is listening for the `server-ready` event.
+
+```js {16-19}
+window.addEventListener('load', async () => {
+  textareaEl.value = files['index.js'].file.contents;
+  textareaEl.addEventListener('input', (e) => {
+    writeIndexJS(e.currentTarget.value);
+  });
+
+  const terminal = new Terminal({
+    convertEol: true,
+  });
+  terminal.open(terminalEl);
+
+  // Call only once
+  webcontainerInstance = await WebContainer.boot();
+  await webcontainerInstance.mount(files);
+
+  // Wait for `server-ready` event
+  webcontainerInstance.on('server-ready', (port, url) => {
+    iframeEl.src = url;
+  });
+});
+```
+
+Open the `loading.html` file and change the message.
+
+```html
+Use the terminal to run a command!
+```
+
+## 2. Start the shell
+
+In order to make the terminal a bit more usuable, instead of spawning commands separately, we could spawn `JSH` instead. 
```

Comment:
Not sure if we have talked about `jsh` before this?

Reply:
Oh yeah, I wanted to ask if we should 

a) Spawn `zsh` instead 🤷 ?
b) Explain what `jsh` is?

Reply:
What do you think, @jrvidal and @SamVerschueren? 
What are the plus/cons of keeping `jsh`? Could we just explain it with one sentence and link external material?

Reply:
I've actually solved it myself 🕵️